Cython/Compiler/Annotate.py

```diff
@@ -108,9 +108,9 @@ def _css(self):
 
     # on-click toggle function to show/hide C source code
     _onclick_attr = ' onclick="{}"'.format((
-        "(function(s){"
-        "    s.display =  s.display === 'block' ? 'none' : 'block'"
-        "})(this.nextElementSibling.style)"
+        "f = this.firstChild, s = this.nextElementSibling.style, "
+        "c = f.nodeValue == '+', s.display = c ? 'block' : 'none', "
+        "f.nodeValue = c ? '-' : '+'"
         ).replace(' ', '')  # poor dev's JS minification
     )
```

Comment:
The function argument was meant to avoid defining global names, which your change now does. Please keep the global namespace clean.